.azuredevops/components/AMDMIGraphX.yml

```diff
@@ -87,7 +87,7 @@ jobs:
         -DCMAKE_CXX_COMPILER=$(Agent.BuildDirectory)/rocm/llvm/bin/clang++
         -DCMAKE_C_COMPILER=$(Agent.BuildDirectory)/rocm/llvm/bin/clang
         -DCMAKE_BUILD_TYPE=Release
-        -DAMDGPU_TARGETS=gfx1030;gfx1100
+        -DAMDGPU_TARGETS=gfx942
```

Comment:
MIGraphX looks to use GPU_TARGETS:
https://github.com/ROCm/AMDMIGraphX/blob/develop/Jenkinsfile#L133

Reply:
Aha, good catch. Let me retest using the correct name

Reply:
Build looks good: https://dev.azure.com/ROCm-CI/ROCm-CI/_build/results?buildId=3963&view=logs&j=b0b0eb5e-9de9-5eb1-7212-04e102eae4f9